crypto/x509/x509_r2x.c

```diff
@@ -37,8 +37,10 @@ X509 *X509_REQ_to_X509(X509_REQ *r, int days, EVP_PKEY *pkey)
             goto err;
         if (!ASN1_INTEGER_set(xi->version, 2))
             goto err;
-/*-     xi->extensions=ri->attributes; <- bad, should not ever be done
```

Comment:
I thought the changes to comments marked with `/*-` were going to be reverted?

Reply:
Why did you mark this question as resolved?

[A comment](https://github.com/openssl/openssl/pull/26675#issuecomment-2683706615) says

> For this pull request, if you revert the `/*-` changes, then it seems okay to proceed

and [you replied](https://github.com/openssl/openssl/pull/26675#issuecomment-2692196386)

> I've already revert the /*- changes, ping for a second review!

but there are two `/*-` that remain, i.e. that haven't been reverted.

Reply:
@tom-cosgrove-arm since I not only remove the - char, but also adjust the comments format, so I delete the - char.

Reply:
So the `/*-` marker means "this is a comment formatted in a non-standard way: please don't reformat it". You were asked not to change them, and said that you had reverted the changes (i.e. put them back the way they were). But actually you are changing them. I would suggest that it's probably best to honour the original author's intentions in a wide-ranging PR such as this and leave them how they were